plugins/modules/elastic_pkgpolicy_info.py

```diff
@@ -70,7 +70,7 @@ def main():
       module.exit_json(**results)
       
     if module.params.get('integration_name'):
-      integration_object = kibana.check_integration(module.params.get('integration_name'))
+      integration_object = kibana.check_integration(integration_name)
```

Comment:
If your switching to using the variable on 73 should you also use it instead of the module param in the check on 72?